main.go

```diff
@@ -94,22 +99,40 @@ func main() {
 		if err != nil {
 			svc.Logger.Fatal(err)
 		}
+		//add default user to db
+		user := &User{}
+		svc.db.FirstOrInit(user, User{AlbyIdentifier: "dummy"})
```

Comment:
guess we could rename that column to `identifier` only.

Reply:
We already have an identifier - it's the `ID` (the primary key).